CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to Spotlight
+
+Spotlight is a collaborative, open source project produced by developers, designers, product owners from several organizations. Spotlight mostly, uses the [contribution model](https://github.com/projectblacklight/blacklight/blob/master/CONTRIBUTING.md) from it's parent project [Blacklight](https://github.com/projectblacklight/blacklight). There are a few differences from the Blacklight process that this document will highlight.
```

Comment:
- remove comma following "mostly"
- "it's parent" should be "its parent" (no apostrophe)